### ISSUE TITLE
bind can be improved for <textarea>

### ISSUE THREAD

**Issue by torhovland** [aspnet/Blazor#434]
I have to do this in order to get the initial value loaded into the textarea:

```
<textarea @bind(myvar)>@myvar</textarea>
```

It would be nicer and more consistent with `<input>` if this would work the same way:

```
<textarea @bind(myvar)></textarea>
```

**Comment by RyoukoKonpaku** [comment]
It should be fixed I guess once PR #406 is merged as they're revamping the bind system to be more flexible and powerful.

**Comment by SteveSandersonMS** [comment]
I think that even after #406 we'll still need to make a tweak in `BrowserRenderer.ts` in the `tryApplyValueProperty` function so that it understands that `textarea` requires special handling for `value` (equivalent to how it already does for `input` and `select`).

This should be a fairly easy tweak (and doesn't need to wait for #406) if anyone's interested in submitting a PR :)

**Comment by RyoukoKonpaku** [comment]
@SteveSandersonMS  Sounds easy enough xD can I give it a shot?

**Comment by SteveSandersonMS** [comment]
Resolved in #439
